badgeteam/mch2022-firmware-esp32
# Update PAX' error reporting to reduce spam.

**WARNING:** It is absolutely **REQUIRED** to verify this pull request by pulling the branch, flashing and testing.
This is because my badge broke.

How to test:
 1. Call `pax_draw_image(your_buffer_here, NULL, 0, 0);` in a constant loop.
 2. Observe as PAX should report `No framebuffer` error every two seconds.
 3. Observe as PAX should report some number of silenced warnings.
